### DISCUSSION TITLE
just migrated: docker compose question

### DISCUSSION THREAD

**Discussion by steveoh** [podman-desktop/podman-desktop#1680] [Q&A]
I just migrated from docker desktop to podman desktop. It looks so nice. I wish I would have done this sooner. Thank you 🎉 

I made a few mistakes when starting.

1. I didn't realize docker desktop had to not be running

I had to then stop it, and use the `podman-mac-helper install`

2. My first test was to use docker compose - since it's quick and easy.

It failed and I found this [blog post](https://onlyutkarsh.medium.com/podman-cannot-connect-to-the-docker-daemon-error-on-macos-f65a6759204) instead of the instructions on 

- https://podman-desktop.io/docs/migrating-from-docker/using-the-docker_host-environment-variable
- https://podman-desktop.io/docs/podman-compose

I followed his instructions and it worked and now I'm not sure if I should try to undo them and follow the official documentation or not. When I did try to set `DOCKER_HOST` from the migration docs, it didn't work. But maybe thats because of the ssh changes I added and don't know how to remove. 😨 

What say the community?

**Comment by mairin** [comment]
@steveoh Thanks for giving Podman Desktop a try!! I am not 100% sure the best path for you, but we have a new feature coming up in the next release that will show you the current state of podman docker compatibility:

https://github.com/containers/podman-desktop/issues/905#issuecomment-1341538249

So that will be something to look forward to in 0.13.0!

Is your set up still working for you now?

**Reply by steveoh** [reply]
It's mostly working. Still having issues with volumes.

**Reply by steveoh** [reply]
```bash
$ pwd 
/Users/steveoh/dev/gis.utah.gov

$ podman run --rm --label=jekyll --volume=./:/srv/jekyll -it -p 4000:4000 jekyll/jekyll:4 jekyll serve --watch --livereload --incremental
Error: statfs /Users/steveoh/dev/gis.utah.gov: no such file or directory
```

**Reply by benoitf** [reply]
I think I noticed at some point that if you don't have some directories when you start the podman machine, then it won't work in the mount steps.

Could you try to restart your podman machine ?

**Comment by cdrage** [comment]
@steveoh We've just integrated this functionality under a "compatibility" button for macOS! So you don't have to go through the tutorial. https://github.com/containers/podman-desktop/pull/1697

It'll be out in the next release. 

*In theory..* after pressing the compatibility button you should be able to use the docker-compose CLI correctly against podman no-problem. 

In practice, there are so many different versions of Docker as well as Podman Desktop that they sometimes conflict.

Once the next release comes out, feel free to try the new compatibility mode function and let us know if it works for you!
